sound/soc/sof/intel/ext_manifest.h

```diff
@@ -21,7 +21,9 @@
 enum sof_cavs_config_elem_type {
```

Comment:
typo in commit: "this elems" -> "these elements". no need to abbreviate in the description.